src/lib/containers/drug_upload_tool/UserFileGrid.tsx

```diff
@@ -69,6 +67,14 @@ export default class UserFileGrid extends React.Component<
       <p>Are you sure you want to trash this submission?' </p>
     </>
   )
+  static requestFilter = {
```

Comment:
If this is unchanging, could you make this a const (not just a single instance associated to the class)

Reply:
TS yells at me that it can't be static and constant but I did make it read only